sdk/tests/js/webgl-test-utils.js

```diff
@@ -3136,38 +3128,30 @@ var runSteps = function(steps) {
  * @param {!function(!HTMLVideoElement): void} callback Function to call when
  *        video is ready.
  */
-function startPlayingAndWaitForVideo(video, callback) {
+async function startPlayingAndWaitForVideo(video, callback) {
   if (video.error) {
     testFailed('Video failed to load: ' + video.error);
     return;
   }
 
-  video.addEventListener(
```

Comment:
I think you still want this, otherwise an error after playback startup won't be caught.

Reply:
Honestly I think it's more likely to cause problems to have errors that might happen later. We just added assertions that errors did not happen after finishTest() is called, and this seems like it risks that. `await video.play()` should throw in any case that we care about here.

Reply:
Possibly, I'd probably at least keep a console log or something then. The risk is only to debuggability in case the playback failure leads to conformance test failures.

Reply:
I feel like this ought to be handled by warnings that the browser generates, or otherwise should instead happen following an unexpected failure in some later texImage(video).